assignment-1/Models/User.cs

```diff
@@ -0,0 +1,14 @@
+public class User
+{
+    public string Name { get; set; }
+    public UserType UserType { get; set; }
+    public List<Book> IssuedBooks { get; set; }
+    public DateTime ReturnDate { get; set; }

```

Comment:
Here as well you have `ReturnDate` as a property of User. Shouldn't this be a property of IssuedBook object?